### ISSUE TITLE
No Accessibility documentation (A11Y)

### ISSUE THREAD

**Issue by aha999** [Chocobozzz/PeerTube#5851]
### Describe the desired information/correction

PeerTube currently doesn't have any publicly available documentation regarding user accessibility, which is troublesome, as it would be very useful for people with disabilities to have a source of information available.

Information about any keyboard shortcuts, WCAG2.1 compatibility, screen reader support etc.

I suggest that a section in documentation is to be made for accessibility.

**Comment by Chocobozzz** [comment]
Hi,

Can you provide an example of such documentation?

**Comment by aha999** [comment]
@Chocobozzz Of course. I've put (edited) examples into the [main comment](https://github.com/Chocobozzz/PeerTube/issues/5851#issue-1773550552).

**Comment by Chocobozzz** [comment]
We plan to implement this feature in 2024. Read the detailed blog post on https://joinpeertube.org/news/peertube-future-2024

**Comment by Chocobozzz** [comment]
Better late than never, here it is: https://docs.joinpeertube.org/use/accessibility
